tests/views_integration/Tribe/Events/Views/V2/Views/Latest_Past_ViewTest.php

```diff
@@ -10,6 +10,13 @@
 class Latest_Past_ViewTest extends ViewTestCase {
 	use With_Post_Remapping;
 
+	/**
```

Comment:
@lucatume The files above related to the change look good. Did you intend to include the changes to the tests for latest past, HTML_cachetest, and iCalendar Request test?

Reply:
Yes, the change is intentional. While working on this, I found out some tests would not behave the same depending on their running alone, in the context of their test case or in the context of the full test suite. The changes address that.